src/main/java/org/opentripplanner/routing/api/request/request/TransitRequest.java

```diff
@@ -30,6 +30,8 @@ public class TransitRequest implements Cloneable, Serializable {
   private List<FeedScopedId> unpreferredRoutes = List.of();
   private DebugRaptor raptorDebugging = new DebugRaptor();
 
+  private boolean enabled = true;
```

Comment:
Why do we need this boolean?

Reply:
Left over from a previous version. I'll remove it.